spec/helpers/capybara_poltergeist_screenshot.rb

```diff
@@ -0,0 +1,12 @@
+require 'launchy'
+
+module Capybara
+  module PoltergeistScreenshot
+    def screenshot_and_open_image(full: false)
```

Comment:
Nope, thanks for pointing that out. I'm accustomed to using the `capybara-screenshot` gem which offers save_and_open_image but only works with capybara-webkit. I'll take this out.